docs/administer/custom_policies.md

```diff
@@ -0,0 +1,122 @@
+        | **Topology Key**  | Specifies the domain used to group nodes or pods for affinity. Determines the scope (e.g., zone, hostname) for applying scheduling rules. |**Examples:**<br> `kubernetes.io/hostname`<br>`topology.kubernetes.io/zone`<br> `topology.kubernetes.io/region`<br>Custom: `rack` |
+        | **Key**           | The pod label key used in **Pod Affinity** or **Anti-Affinity** rules. Helps target specific pods to influence scheduling decisions. | Should match a label present on existing pods in the cluster.<br>**Examples:**<br>- `app`<br>- `security`<br>- `environment`<br>- Custom: `web-store` |
+        | **Operator**      | Logical condition used to evaluate the **Key** and **Values**. Determines how Kubernetes interprets the label match. | **Supported Operators:**<br>- `In`: Matches if the label value is in a specified list<br>- `NotIn`: Matches if not in the list<br>- `Exists`: Matches if the label key exists (regardless of value)<br>- `DoesNotExist`: Matches if the label key does not exist |
+        | **Values**        | Specific label values that must match for the rule to apply. Required when using `In` or `NotIn` operators. | **Examples:**<br>- `s2`<br>- `database`<br>- `production`<br>- Custom: `finance`, `cache-tier` |
+
+6. Click **Add** to save the rule. The new pod scheduling policy is now available and can be applied to relevant components.
+
+## Update Pod scheduling rule for a policy
+
+!!! warning
+    When you update the policy rules, the database cluster may restart to apply the new affinity configuration.
+
+Here are the steps to update a Pod scheduling policy rule:
+{.power-number}
+
+1. From the Percona Everest home page, navigate to the <i class="uil uil-cog"></i> **Settings > Pod scheduling policies** page.
+
+2. Select the policy that you want to update.
+
+3. Click on the **Edit** icon. The **Edit rule** pop-up opens.
+
+    ![!image](../images/edit_pod_scheduling.png)
+
+4. Make the required changes and click **Save**. 
+
+    The updated policy will be applied to all the affected database clusters.
+
+## Delete pod scheduling policy
+
+If a specific Pod scheduling policy is no longer needed, the Percona Everest Administrator or anyone with the proper permissions can delete it.
+
+Here are the steps to delete a Pod scheduling policy:
+{.power-number}
+
+1. From the Percona Everest home page, navigate to the <i class="uil uil-cog"></i> **Settings > Pod scheduling policies** page.
+
+2. Click the **ellipsis** (three dots) next to the policy you want to delete.
+
+3. Click on the **Delete** icon. The **Delete Rule** confirmation pop-up opens.
+
+4. Click **Delete**.
+
+    !!! note
+        If a policy is applied to any cluster, you must first [remove the policy](apply_policies_cluster.md#change-pod-scheduling-policy-for-an-existing-db-cluster) before you can delete it.
```

Comment:
this warning looks in the following way:

**_if you want to remove a policy, you must first delete the policy._**

It really confuses the reader.

Reply:
Fixed.